### PR TITLE
Added documentation for ROVA sensor

### PR DESCRIPTION
**Pull request by GidoHakvoort**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18868

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.rova.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+### {% linkable_title Full configuration sample %}
+
+A full configuration entry would look like the sample below.
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: solaredge
```
Comment (gerard33) [inline source/_components/sensor.rova.markdown]:
```suggestion
  - platform: rova
```

Comment (GidoHakvoort) [reply source/_components/sensor.rova.markdown]: Thanks, missed this during copy/paste from another pull request.

File: source/_components/sensor.rova.markdown [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+ha_iot_class: "Cloud Polling"
+---
+
+[ROVA](https://rova.nl) is a waste collection company that operates in the center and east of the Netherlands. The `rova` platform uses an unofficial [ROVA](https://rova.nl) API to allow you to get your waste collection schedule and integrate this in you Home Assistant installation.
+
+## {% linkable_title Configuration %}
+
+To use the ROVA sensor in your installation, add the following to your configuration.yaml file:
+
+{% raw %}
```
Comment (frenck) [inline source/_components/sensor.rova.markdown]: There is no need for the raw escape here?

File: source/_components/sensor.rova.markdown [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rova
+    zip_code: ZIP_CODE
+    house_number: HOUSE_NUMBER
+    monitored_conditions:
```
Comment (frenck) [inline source/_components/sensor.rova.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.rova.markdown [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+      description: Upcoming collection date of your general waste
+{% endconfiguration %}
+
+If no **monitored_conditions** are specified, only **gft** will be enabled.
+
+### {% linkable_title Full configuration sample %}
+
+A full configuration entry would look like the sample below.
+
+{% raw %}
```
Comment (frenck) [inline source/_components/sensor.rova.markdown]: No need to raw escape here.

File: source/_components/sensor.rova.markdown [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+comments: false
+sharing: true
+footer: true
+logo: rova.jpg
+ha_category: Sensor
+ha_release: 
+ha_iot_class: "Cloud Polling"
+---
+
+[ROVA](https://rova.nl) is a waste collection company that operates in the center and east of the Netherlands. The `rova` platform uses an unofficial [ROVA](https://rova.nl) API to allow you to get your waste collection schedule and integrate this in you Home Assistant installation.
```
Comment (frenck) [inline source/_components/sensor.rova.markdown]:
```suggestion
[ROVA](https://rova.nl) is a waste collection company that operates in the center and east of the Netherlands. The `rova` platform uses an unofficial [ROVA](https://rova.nl) API to allow you to get your waste collection schedule and integrate this in your Home Assistant installation.
```

File: source/_components/sensor.rova.markdown [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+title: "ROVA Sensor"
+description: "Instructions on how to integrate ROVA sensor within Home Assistant."
+date: 2018-12-08 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rova.jpg
+ha_category: Sensor
+ha_release:
```
Comment (frenck) [inline source/_components/sensor.rova.markdown]: Please provide a release number. Next release is recommended. 0.85 at the moment of writing this.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @GidoHakvoort! I've reviewed it, and left you some comments.

**Comment by GidoHakvoort** [comment]
@frenck Nice, thanks for the reviews frenck (both doc and platform). I've updated the doc to resolve the issues.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by klaasnicolaas** [comment]
I see that the parent PR has been merged. So time to do that with this PR also 😄 Thnx for the PR @GidoHakvoort 👍

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**